frontend/src/app/aethel/aethel.component.ts

```diff
@@ -88,12 +95,28 @@ export class AethelComponent implements OnInit {
                 const limit = query["limit"]
                     ? parseInt(query["limit"], 10)
                     : 10;
+                const sort = query["sort"] ?? "";
 
-                this.form.patchValue({ word, type, skip, limit });
-                this.apiService.input$.next({ word, type, skip, limit });
+                this.form.patchValue({ word, type, skip, limit, sort });
+                this.apiService.input$.next({ word, type, skip, limit, sort });
             });
     }
 
+    private initializeSort(): void {
+        const currentSortValue = this.form.controls.sort.value;
+        if (!this.datatable) {
+            return;
+        }
+        this.datatable.sortField =
+            currentSortValue === "" ? null : currentSortValue.replace("-", "");
+        this.datatable.sortOrder =
```

Comment:
I like the ternary operator, but nesting it twice is a bit much IMO

Reply:
😂